integration-tests/elytron-resteasy-reactive/src/main/java/io/quarkus/it/resteasy/reactive/elytron/FruitResource.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.it.resteasy.reactive.elytron;
+
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import io.quarkus.resteasy.reactive.jackson.SecuredSerialization;
+
+@Path("fruit")
+public class FruitResource {
+
+    private final List<Fruit> fruits = List.of(Fruit.APPLE, Fruit.PINEAPPLE);
+
+    @GET
+    @Path("all-no-security")
+    public List<Fruit> allNoSecurity() {
+        return fruits;
+    }
+
+    @SecuredSerialization
```

Comment:
The way I have things now, this is mandatory (the security feature is opt-in only). Moreover it needs to be added to each method separately.

I opted to make this restrictive to start out, but I am not sure if it's a great experience...

Reply:
May be applying it at a class level would apply to all the methods ?